ros/src/pybullet_ros/plugins/body_vel_control.py

```diff
@@ -88,26 +89,26 @@ def cmd_vel_force_ctrler(self):
         experimental code to set a torque on the robot model using a PID controller
         """
         pass
-        #self.controller[0].setpoint = self.cmd_vel_msg.linear.x
-        #self.controller[1].setpoint = self.cmd_vel_msg.linear.y
-        #self.controller[2].setpoint = self.cmd_vel_msg.linear.z
-        #self.controller[3].setpoint = self.cmd_vel_msg.angular.x
-        #self.controller[4].setpoint = self.cmd_vel_msg.angular.y
-        #self.controller[5].setpoint = self.cmd_vel_msg.angular.z
+        # self.controller[0].setpoint = self.cmd_vel_msg.linear.x
```

Comment:
if you don't need / use this controller, it would be better to remove it entirely. you have the original source in git history if you want it back